pages/builds/environment_variables.md.erb

```diff
@@ -153,108 +157,118 @@ The following environment variables are automatically provided to every job, and
 
 You can define environment variables in your jobs in a few ways, depending on the nature of the value being set:
 
-* Pipeline settings — for values that are not secret.
+* Pipeline settings via the Buildkite dashboard — for values that are not secret.
 * [Build pipeline configuration](/docs/pipelines/command-step) — for values that are not secret.
-* An `environment` or `pre-command` [agent hook](/docs/agent/v2/hooks) — for values that are secret or agent-specific.
-
-## Environment variable precedence
-
-You can set environment variables in lots of different places, and which ones take precedence can get a little confusing.
-There are many different levels at which environment variables are merged together. The following walkthrough and examples demonstrate the order in which variables are combined, as if you had set variables in every available place.
+* An `environment` or `pre-command` [agent hook](/docs/agent/v3/hooks) — for values that are secret or agent-specific.
 
-When a job runs on an agent, the first combination of environment variables happens in the job itself:
+### Environment variable precedence
```

Comment:
Is this supposed to be underneath the "Defining your own" section? I feel like it makes more sense as its own section.